lenskit-eval/build.gradle

```diff
@@ -39,6 +39,7 @@ apply from: "$rootDir/gradle/test-utils.gradle"
 
 dependencies {
     compile project(':lenskit-core')
+    compile project(':lenskit-knn')
```

Comment:
I'd rather not have the eval module grow a dependency on kNN. Fortunately, there are two easy ways of fixing this:

- Change the `knn` dependency from `compile` to `testCompile` (so it's only pulled in for the tests)
- Alter the tests to use a baseline scorer instead of a k-NN scorer (e.g. `ItemMeanRatingItemScorer`)

Either one would be acceptable.